modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCSSIncludeMediaQueryCheck.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeCSSIncludeMediaQueryCheck extends BaseUpgradeCheck {
+
+	@Override
+	protected String format(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		Matcher matcher = _includeMediaQueryPattern.matcher(content);
+
+		if (matcher.find()) {
+			String message = StringBundler.concat(
+				"Do not use 'include media-query', do replace by equivalent ",
```

Comment:
Hi @TamyrisTorres, please could you update the message to: "Do not use 'media-query' mixing, replace with its equivalent (e.g., media-breakpoint-up, media-breakpoint-only, media-breakpoint-down, etc.), see LPS-194507."